src/utils.ts

```diff
@@ -57,18 +57,85 @@ export function toEngineering(value: Decimal): Decimal {
   );
 }
 
+/**
+ * Transforms 1-digit mantissa decimal into 6-digits mantissa decimal
+ * For example: 1.5e10 => 15000e3
+ * Note, that it does so in 6 exponent increments, so 1.5e6 is still 1.5e6
+ */
+export function toLongScale(value: Decimal): Decimal {
+  // Give numbers between a thousand and a million exponent 3.
+  let mod = value.exponent < 6 ? 3 : 6;
```

Comment:
Codacy found an issue: [Identifier 'mod' is never reassigned; use 'const' instead of 'let'.](https://app.codacy.com/gh/antimatter-dimensions/notations/pullRequest?prid=6218257)